docs/getting-started/getting-started.md

```diff
@@ -585,7 +584,7 @@ public class MainActivity extends AppCompatActivity {
 
 To complete the tutorial, add the `MovieDetailsMiniApp` to the application.
 
-We've developed and published this MiniApp so you can reuse it for this tutorial. If you want to view the code of the MiniApp, you can view it in [this GitHub repository](https://github.com/electrode-io/MovieDetailsMiniApp)
+We've developed and published this MiniApp to reuse it in this tutorial. You may view the source code of [MovieDetailsMiniApp](https://github.com/electrode-io/MovieDetailsMiniApp) repo.
```

Comment:
I prefer your wording. But for the second phrase would use instead something like `You may view the code of this MiniApp in the [MovieDetailsMiniApp repository](https://github.com/electrode-io/MovieDetailsMiniApp)`